examples/openspending/pages/blog.tsx

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs';
+import React from 'react';
+import { GetStaticProps } from 'next';
+import { BlogsList, SimpleLayout } from '@flowershow/core';
+import clientPromise from '../lib/mddb';
+import type { CustomAppProps } from './_app';
+import Layout from '@/components/_shared/Layout';
+
+interface BlogIndexPageProps extends CustomAppProps {
+  blogs: any[]; // TODO types
+}
+
+export default function Blog({
+  blogs,
+  meta: { title, description },
+}: BlogIndexPageProps) {
+  return (
+    <Layout>
+      <div className="blog-list">
+        <SimpleLayout title={title} description={description}>
+          <BlogsList blogs={blogs} />
+        </SimpleLayout>
+      </div>
+    </Layout>
+  );
+}
+
+export const getStaticProps: GetStaticProps = async () => {
+  const mddb = await clientPromise;
+  const blogFiles = await mddb.getFiles({ folder: 'blog' });
+
+  const blogs = blogFiles.map((item) => ({
+  _id: item._id,
+  file_path: item.file_path,
+  urlPath: item.url_path,
+  date: item.url_path.split('/').slice(-1)[0].split('-').slice(0,3).join('-'),
+    ...item.metadata
+  }));
+
```

Comment:
Blog posts must be sorted by date

![image](https://github.com/datopian/portaljs/assets/16550934/bc0b633f-31cd-4c30-ae80-ace02dfdf559)

Reply:
Done

Reply:
Seems like it's still broken

![image](https://github.com/datopian/portaljs/assets/16550934/4599c6be-7151-4acf-ba25-ef150b2e78be)

![image](https://github.com/datopian/portaljs/assets/16550934/5993d6e2-80fe-4ea3-bf16-21aaafb73060)